integrations/appboy/lib/index.js

```diff
@@ -307,6 +307,17 @@ Appboy.prototype.identify = function(identify) {
     delete traits[key];
   }, reserved);
 
+  // Remove nested hash objects as Braze only supports nested array objects in identify calls
+  // https://segment.com/docs/destinations/braze/#identify
+  each(function(value, key) {
+    if (
+      typeof value === 'object' &&
+      Object.prototype.toString.call(value) !== '[object Array]'
```

Comment:
could you use `Array.isArray()`?

Reply:
I could, but this also supports older browsers and I don't think there is any polyfilling done for these integrations

Reply:
We have several other integrations that use `Array.isArray()`, so it should be safe to do so here.

Reply:
```suggestion
      Array.isArray(value)
```